### DISCUSSION TITLE
关于data.db的疑问

### DISCUSSION THREAD

**Discussion by lxl66566** [Planshit/Tai#66] [Q&A]
早期data.db中，表 `DailyLogModels` 和 `HoursLogModels` 的 `AppModelID` 存在值为 `0` 的记录，此值无法与`AppModels`中的任一ID对应，请问这些 0 该如何解释？
data.db.version: 1.0.0.2
![image](https://user-images.githubusercontent.com/88281489/194748150-e6b6252f-d848-4496-9adb-03729d4bbb0a.png)

**Comment by noberumotto** [comment]
早期版本没有AppModels表，当时是把进程名称和使用时长数据都记录在DailyLogModels表里，后来更新加入了AppModels表，才有了AppModelID这个字段。而这些APPID为0的记录可能是旧版本升级后数据没有成功迁移产生的，总之就是不必理会。

**Reply by lxl66566** [reply]
感谢解答
